nnpdf_data/nnpdf_data/theory.py

```diff
@@ -36,58 +44,78 @@ class TheoryCard:
     mt: float  # # [GeV] top mass
     Qmt: float  # [GeV] MSbar mass reference scale of the top
     ktThr: float  # Threshold ratio of the top
-    CKM: list[float]  # CKM matrix elements (running on the columns first, i.e. CKM_11 is CKM[0] and CKM_12 is CKM[1] and so on)
+    CKM: list[
```

Comment:
I understand this comes from `black` but I would say that it is not the best formatting ever.